examples/aws-nodejs/index.js

```diff
@@ -79,16 +80,46 @@ function getSTSClient () {
 app.use(bodyParser.urlencoded({ extended: true }), bodyParser.json())
 
 app.get('/', (req, res) => {
+  res.setHeader('Content-Type', 'text/html')
   const htmlPath = path.join(__dirname, 'public', 'index.html')
   res.sendFile(htmlPath)
 })
+app.get('/index.html', (req, res) => {
+  res.setHeader('Location', '/').sendStatus(308).end();
+})
+app.get('/withCustomEndpoints.html', (req, res) => {
+  res.setHeader('Content-Type', 'text/html')
+  const htmlPath = path.join(__dirname, 'public', 'withCustomEndpoints.html')
+  res.sendFile(htmlPath)
+})
+
+app.get('/uppy.min.mjs', (req, res) => {
+  res.setHeader('Content-Type', 'text/javascript')
+  const bundlePath = path.join(__dirname, '../..', 'packages/uppy/dist', 'uppy.min.mjs')
+  if (existsSync(bundlePath)) {
+    res.sendFile(bundlePath)
+  } else {
+    console.warn('No local JS bundle found, using the CDN as a fallback. Run `corepack yarn build` to make this warning disappear.')
+    res.end('export * from "https://releases.transloadit.com/uppy/v4.0.0-beta.10/uppy.min.mjs";\n');
+  }
+})
+app.get('/uppy.min.css', (req, res) => {
+  res.setHeader('Content-Type', 'text/css')
+  const bundlePath = path.join(__dirname, '../..', 'packages/uppy/dist', 'uppy.min.css')
+  if (existsSync(bundlePath)) {
+    res.sendFile(bundlePath)
+  } else {
+    console.warn('No local CSS bundle found, using the CDN as a fallback. Run `corepack yarn build` to make this warning disappear.')
+    res.end('@import "https://releases.transloadit.com/uppy/v4.0.0-beta.10/uppy.min.css";\n');
+  }
+})
```

Comment:
Would be nice to make this file less overwhelming by keeping all of this on the client. I don't see a need to add it here?

Reply:
It's useful to test local changes – case in point, in this instance the CDN does not support `endpoint` option, making it impossible to test. A better approach would be to move those to a separate file, or maybe use comments to separate them from the rest of the example

Reply:
What about using `workspace:^` to use the local packages in the example?

Reply:
Yeah I figured Vite is so minimal to setup and it keeps in sync with latest changes, instead of manually pushing build files or depending on published versions.